src/swap/SwapScreenV2.tsx

```diff
@@ -0,0 +1,1149 @@
+    handleSelectPercentageAmount(percentage)
+    setSelectedPercentage(percentage)
+
+    if (!fromToken) {
+      // Should never happen
+      return
+    }
+    AppAnalytics.track(SwapEvents.swap_screen_percentage_selected, {
+      tokenSymbol: fromToken.symbol,
+      tokenId: fromToken.tokenId,
+      tokenNetworkId: fromToken.networkId,
+      percentage,
+    })
+  }
+
+  function onPressLearnMore() {
+    AppAnalytics.track(SwapEvents.swap_learn_more)
+    navigate(Screens.WebViewScreen, { uri: links.swapLearnMore })
+  }
+
+  function onPressLearnMoreFees() {
+    AppAnalytics.track(SwapEvents.swap_gas_fees_learn_more)
+    navigate(Screens.WebViewScreen, { uri: links.transactionFeesLearnMore })
+  }
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer} testID="SwapScreen">
+      <CustomHeader
+        style={{ paddingHorizontal: variables.contentPadding }}
+        left={<BackButton />}
+        title={t('swapScreen.title')}
+      />
+      <ScrollView
+        contentContainerStyle={styles.contentContainer}
+        keyboardShouldPersistTaps="handled"
+      >
+        <View style={{ display: 'flex', justifyContent: 'space-between', flexGrow: 1 }}>
+          <View style={{ flexShrink: 0 }}>
+            <View style={[styles.warningsContainer, { gap: 4 }]}>
+              <TokenEnterAmount
+                autoFocus
+                token={fromToken}
+                inputValue={amountFrom}
+                inputRef={inputFromRef}
+                tokenAmount={processedAmountsFrom.token.displayAmount}
+                localAmount={processedAmountsFrom.local.displayAmount}
+                onInputChange={handleAmountInputChange}
+                amountType={amountType}
+                toggleAmountType={handleToggleAmountType}
+                onOpenTokenPicker={onOpenTokenPickerFrom}
+                testID="SwapAmountInput"
+              />
+              <View style={styles.switchTokensContainer}>
+                <Touchable
+                  borderless
+                  borderRadius={Spacing.Regular16}
+                  shouldRenderRippleAbove
+                  style={styles.switchTokens}
+                  onPress={handleSwitchTokens}
+                  testID="SwapScreen/SwitchTokens"
+                >
+                  <CircledIcon radius={Spacing.Large32} backgroundColor={colors.black}>
+                    <ArrowDown color={colors.white} />
+                  </CircledIcon>
+                </Touchable>
+              </View>
+              <TokenEnterAmount
+                token={toToken}
+                inputValue={amountTo}
+                inputRef={inputToRef}
+                tokenAmount={processedAmountsTo.token.displayAmount}
+                localAmount={processedAmountsTo.local.displayAmount}
+                amountType={amountType}
+                onOpenTokenPicker={onOpenTokenPickerTo}
+                loading={shouldShowSkeletons}
+                testID="SwapAmountInput"
```

Comment:
this `testID` is identical to the "from" component. perhaps a typo?

Reply:
@bakoushin its per design. It was like that before rework and I've kept it the same cause in the test files it searches for the same `testID`s but in different containers (`from/to`)